web/src/app/views/about-view/about-view.component.scss

```diff
@@ -17,3 +17,11 @@
     display: block;
   }
 }
+
+@import "~styles/variables";

```

Comment:
Tohle tam není potřeba, když nepoužíváš žádné proměnné z toho importovaného souboru.

Reply:
Jo, to jsem si nebyl jistý, tak jsem to tam radši zkopíroval taky, ale už to dávám pryč.